Plugin~/WebRTCPlugin/Codec/EncoderFactory.cpp

```diff
@@ -8,6 +8,7 @@
 
 #if defined(SUPPORT_D3D11)
 #include "NvCodec/NvEncoderD3D11.h"
+#include "SoftwareCodec/SoftwareEncoder.h"

```

Comment:
SoftwareEncoder by itself only depends on the base class and does not depend on the graphics implementation class (D3D11, etc), so I think we can put it outside ifdef